src/Microsoft.Data.DataView/SchemaDebuggerProxy.cs

```diff
@@ -50,7 +50,7 @@ public AnnotationsDebuggerProxy(DataViewSchema.Annotations annotations)
         private static object GetValue<T>(DataViewSchema.Annotations annotations, int columnIndex)
         {
             T value = default;
-            annotations.GetGetter<T>(columnIndex)(ref value);
+            annotations.GetGetter<T>(annotations.Schema[columnIndex])(ref value);
```

Comment:
This method is only called in one place - which already has the Column. Can we just pass the column through here?